murat_reis/Orta_Algoritma_2.py

```diff
@@ -1,34 +1,47 @@
 import os
 from cryptography.fernet import Fernet
 
-fname = "passwords.txt"
-key = Fernet.generate_key()
-wee = Fernet(key)
+fname = "passwords.txt"  # Kayıt dosyası adı
```

Comment:
1\. modül için `passwords.txt` dosyası kullanılırken 2. modül için de yeni bir dosya oluşturup `passwords_encrypted.txt` diyebilirsin, böylece modüller aynı dosya üzerinde çakışmaz. O şekilde PR daha temiz olur.